Academia-750/academia750-frontend-vue
# Frontend List of groups

## References


## Context

This PR contains The UI designs for the group listing table but it's now integrated with the students API which will be replaced as soon as I get the groups API.



## How To Test

- Login As an admin
- on the sidebar check the Group management

- Login as a user
- got to your account settings
- click on rewards
- and in in rewads go to referrals

### Test Set Up

### Test Steps

### Platforms

## Screen Shoots
